src/components/spinner/index.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import Spinner from './index.tsx';
+
+<Meta
+  title="Components/Feedback/Spinner"
+  component={Spinner}
+  args={{
+    size: 'md',
+  }}
+  argTypes={{
+    size: {
+      options: ['xs', 'sm', 'md', 'lg'],
+      control: { type: 'select' },
+    },
+  }}
+/>
+
+export const Template = (args) => {
+  return <Spinner {...args} />;
+};
+
+# Stack
+
+## Overview
```

Comment:
is it possible to add the sizes variations as well? I know is in the args table but it will be much better for visibility on the docs tab.

Reply:
Like we do for the  button:

![Screenshot 2022-11-03 at 11 50 12](https://user-images.githubusercontent.com/37380787/199702345-80e8e98c-8a5f-4e3f-b751-008efd409028.png)

Reply:
Sure